.github/workflows/builds.yml

```
@@ -262,7 +264,9 @@ jobs:
           lfs: true
 
       - name: Set dependencies URL
-        run: echo "DEPENDENCY_URL=https://acts.web.cern.ch/ACTS/ci/ubuntu-22.04/deps.${DEPENDENCY_TAG}.tar.zst" >> $GITHUB_ENV
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:122: Double quote to prevent globbing and word splitting

(shellcheck)